app/src/main/java/com/projectit3048c/MainActivity.kt

```diff
@@ -165,50 +165,46 @@ class MainActivity : ComponentActivity() {
             )
             Button(
                 onClick = {
-                    var specimen = FoodAmount().apply {
+                    var foodAmount = FoodAmount().apply {
                         foodName = inFoodName
-                        foodId = selectedFood?.let(){
-                            it.id
-                        } ?: 0
+                        foodId = selectedFood?.id ?: 0
                         foodAmount = inAmount
                         foodIntake = inIntake
                         foodLoged = inLoged
                     }
-                    Toast.makeText(
-                        context,
-                        "$inFoodName $inAmount $inIntake $inLoged",
-                        Toast.LENGTH_LONG)
-                        .show()
+                    Toast.makeText(context, foodAmount.toString(), Toast.LENGTH_LONG).show()
                 }
             ) {
                 Text(text = "Add")
             }
         }
+    }
 
-        fun delete(foodItems: FoodItems) {
-            //  ViewModel.deleteSavedFoodDatabase(foodItems)
-        }
+    fun delete(foodItems: FoodItems) {
+        //  ViewModel.deleteSavedFoodDatabase(foodItems)
+    }
```

Comment:
1) While you're here, remove commented out code.  Avoid pushing commented code to Version Control.  Version control gives us better options:
- You can look at history to see what used to be there.
- You can use branches for experimental features.
2) Formatting code can (and should) be performed automatically, and consistently, by the IDE. Formatting code is really programmer's preference anyway, and doesn't necessarily decrease technical debt.